getMap.js

```diff
@@ -29,7 +30,7 @@ function getMap() {
           callback(null, response);
           debug("Request for " + req.url + " complete in " + (Date.now() - start) + "ms");
         });
-      } else {
+      } else if (protocol == "http" || protocol == "https") {
```

Comment:
It is best to use `===` for strict comparisons 
```suggestion
      } else if (protocol === "http" || protocol === "https") {
```